jenkins/custom-ci/generic/cortx-cc.groovy

```diff
@@ -0,0 +1,114 @@
+        build_upload_dir = "$release_dir/github/integration-custom-ci/$os_version/$release_tag/cortx_iso"
+        python_deps = "${THIRD_PARTY_PYTHON_VERSION == 'cortx-2.0' ? "python-packages-2.0.0-latest" : THIRD_PARTY_PYTHON_VERSION == 'custom' ?  "python-packages-2.0.0-custom" : "python-packages-2.0.0-stable"}"
+    }
+    
+    
+    
+    options {
+        timeout(time: 30, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '5', numToKeepStr: '10'))
+    }
+
+    stages {
+    
+        stage('Checkout cortx-cc') {
+            when { expression { params.BUILD_LATEST_CORTX_CC == 'yes' } }
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                dir ('cortx-cc') {
+                    checkout([$class: 'GitSCM', branches: [[name: "${CORTX_CC_BRANCH}"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CloneOption', depth: 0, noTags: false, reference: '', shallow: false, timeout: 15], [$class: 'SubmoduleOption', disableSubmodules: false, parentCredentials: true, recursiveSubmodules: true, reference: '', trackingSubmodules: false, timeout: 15]], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: "${CORTX_CC_URL}"]]])
+                }
+            }
+        }
+    
+        stage('Install Dependencies') {
+            when { expression { params.BUILD_LATEST_CORTX_CC == 'yes' } }
+            steps {
+                script { build_stage = env.STAGE_NAME }
+
+                sh label: 'Install cortx-cc pre-requisites', script: '''
+                    yum -y install python3 python3-devel facter yum-utils
+                    yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
+                    yum -y install consul-1.9.1
+                '''
+
+                sh label: 'Install cortx-prereq', script: '''
+                    CORTX_UTILS_REPO_OWNER=$(echo $CORTX_UTILS_URL | cut -d "/" -f4)
+                    yum erase python36-PyYAML -y
+                    yum install -y gcc python3 python3-pip python3-devel python3-setuptools openssl-devel libffi-devel python3-dbus
+                    pip3 install  --no-cache-dir --trusted-host cortx-storage.colo.seagate.com -i http://cortx-storage.colo.seagate.com/releases/cortx/third-party-deps/python-deps/$python_deps/ -r https://raw.githubusercontent.com/${CORTX_UTILS_REPO_OWNER}/cortx-utils/${CORTX_UTILS_BRANCH}/py-utils/python_requirements.txt -r https://raw.githubusercontent.com/${CORTX_UTILS_REPO_OWNER}/cortx-utils/${CORTX_UTILS_BRANCH}/py-utils/python_requirements.ext.txt
+                    rm -rf /etc/pip.conf
+                '''
+
+                sh label: 'Install Dependencies', script: '''
+                    set +x
+                    yum-config-manager --add-repo=http://cortx-storage.colo.seagate.com/releases/cortx/components/github/main/$os_version/dev/motr/last_successful/
```

Comment:
For custom-ci we should always use Motr packages generated as part of that custom-ci build.  Refer - https://github.com/Seagate/cortx-re/blob/main/jenkins/custom-ci/generic/hare.groovy#L81

Reply:
Done

Reply:
It's still configuring `main` branch repo - https://github.com/gauravchaudhari02/cortx-re/blob/cortx-cc-custom-ci/jenkins/custom-ci/generic/cortx-cc.groovy#L76